### PR TITLE
change defaults

### PR DESCRIPTION
**Pull request by Jonahss**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,8 +22,11 @@ var defaults = {
 
 var methods = ['get', 'post', 'put', 'patch', 'head', 'delete']
 
-function EazeClient (baseUrl) {
+function EazeClient (baseUrl, options) {
   baseUrl = baseUrl || ''
+  if (options) {
+    defaults = extend(defaults, options)
```
Comment (bendrucker) [inline index.js]: assigning to `defaults` is incorrect here, that changes the value for *all* instances (side effect)

`options = extend(defaults, options)` would work perfectly here

**Comment by bendrucker** [comment]
Why's this necessary? What defaults would someone intend to set on a client level?

**Comment by Jonahss** [comment]
timeout. I've been playing in a repl and when the C# server is breakpointed in the middle of a request the client times out relatively quickly.

File: index.js (hunk 1 above)
Comment (Jonahss) [inline index.js]: good point.  updated pr

**Comment by bendrucker** [comment]
K, makes sense. In that case I'd like to do a breaking release that switches to using a single `options` argument where `base` is an option.

**Comment by Jonahss** [comment]
makes sense, you want me to do that?

**Comment by bendrucker** [comment]
Sure!

**Comment by Jonahss** [comment]
how's that?

**Comment by bendrucker** [comment]
looks good, i'll double check and merge

**Pull request closed without merging**